pkg/mixin/delete.go

```diff
@@ -0,0 +1,25 @@
+package mixin
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+)
+
+type DeleteOptions struct {
+	Name string
+}
+
+func (o *DeleteOptions) Validate(args []string) error {
+	switch len(args) {
+	case 0:
+		// later on this may be okay (porter mixin install --feed deislabs) but not now
```

Comment:
Looks like this comment was copied over from the InstallOptions Validation function but doesn't really apply here. Would you please remove it?